client.go

```diff
@@ -157,3 +157,22 @@ func (c Client[Resource]) List(ctx context.Context, options Options) ([]Resource
 	}
 	return resources, nil
 }
+
+// Create performs a POST request to create a new resource on the target crud. Returns the
+// identifier of the created resource and any error that occurred.
+func (c Client[Resource]) Create(ctx context.Context, resource Resource, options Options) (string, error) {
+	req, err := c.client.NewRequestWithContext(ctx, http.MethodPost, "", resource)
+	if err != nil {
+		return "", err
+	}
+
+	if err := options.setOptionsInRequest(req); err != nil {
+		return "", err
+	}
+
+	createdResource := make(map[string]string)
```

Comment:
Can you define a struct of the expected response?

Reply:
Of course!